modules/monitoring-granting-user-permissions-using-cli.adoc

```diff
@@ -14,24 +14,24 @@ This procedure shows how to grant users permissions for monitoring their own ser
 
 .Procedure
 
-* Run this command to assign <role> to <user> in <namespace>:
+* Run this command to assign a role to a user in a defined namespace:
 +
 [source,terminal]
 ----
 $ oc policy add-role-to-user <role> <user> -n <namespace>
 ----
 +
-Substitute <role> with `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.
+Substitute `<role>` with `monitoring-rules-view`, `monitoring-rules-edit`, or `monitoring-edit`.
 +
 --
-** `monitoring-rules-view` allows reading PrometheusRule custom resources within the namespace.
-** `monitoring-rules-edit` allows creating, modifying, and deleting PrometheusRule custom resources matching the permitted namespace.
-** `monitoring-edit` gives the same permissions as `monitoring-rules-edit`. Additionally, it allows creating new scraping targets for services or Pods. It also allows creating, modifying, and deleting ServiceMonitors and PodMonitors.
+** `monitoring-rules-view` allows reading `PrometheusRule` custom resources within the namespace.
+** `monitoring-rules-edit` allows creating, modifying, and deleting `PrometheusRule` custom resources matching the permitted namespace.
+** `monitoring-edit` gives the same permissions as `monitoring-rules-edit`. Additionally, it allows creating new scraping targets for services or pods. It also allows creating, modifying, and deleting `ServiceMonitor` and `PodMonitor` resources.
 --
 +
 [IMPORTANT]
 ====
 Whichever role you choose, you must bind it against a specific namespace as a cluster administrator.
```

Comment:
I think that this information should be mentioned before you run the command. I'd probably make the cluster admin permissions a prereq and mention that you're granting permission to services by namespace in the intro.